validation.py

```diff
@@ -0,0 +1,245 @@
+import pandas as pd
+import numpy as np
+import re as re
+from models.IntakeRow import RowNames
+
+'''Regular expression translated: Any amount of integers but at least one, a space, a 1-2 letter word (for directions N, SW, etc)
+    Followed by a space, followed by any number of letters/numbers (but at least one) followed by a space. 
+    This is followed by a space, followed by a word (for the St., Ave., etc).
+    This is difficult to maintain though!! Holy cow!'''
+addressRegex = r'^(\d+)\s([a-zA-Z]{1,2})\s([a-zA-Z0-9]+\s)([a-zA-Z]+)'
+addressWithFacilityRegex = r'^(\d+)\s([a-zA-Z]{1,2})\s(([a-zA-Z1-9]+\s)+)([a-zA-Z]+)\s(#\d+)'
+POBoxRegex = r'^([P|p][O|o])\s(Box|box)\s(\d+)(\,)*(\s)[a-zA-Z1-9]+(\,)*\s[A-Z]{2}(\,)*\s\d{5}'
```

Comment:
Looks like this will flag BOX  as invalid.  Are we sure we want this as invalid?